source/js/buyers-guide/components/creep-vote/creep-vote.jsx

```diff
@@ -159,6 +160,18 @@ export default class CreepVote extends React.Component {
    * @returns {jsx} What users see when they have voted on this product.
    */
   renderDidVote(){
+    const NUM_GROUPS = CREEPINESS_LABELS.length;
+    let userVoteGroup = Math.floor(this.state.creepiness/(100/NUM_GROUPS));
+    let creepType;
+
+    if (userVoteGroup < Math.floor(NUM_GROUPS/2)) { // lower half groups
```

Comment:
Can't this be something like

`creepType = CREEPINESS_LABELS[userVoteGroup]`?

Reply:
Nah, it's not 1-1. We only have 3 versions of keyword we wanna share in social share.

Reply:
> version depending on a users vote: "CREEPY" (first two bars), "NOT REALLY CREEPY" (middle), "NOT CREEPY" (last two bars)]

Reply:
We can do what Pomax did to make things consistent.

Reply:
k I'll update - @kristinashu should I make the label all caps in social share? e.g., SOMEWHAT CREEPY, A LITTLE CREEPY etc

Reply:
lol yeah uppercase is great, thank you!

Reply:
Unless that's more complicated, then lowercase is cool too!